oak_functions/examples/key_value_lookup/module/src/tests.rs

```diff
@@ -76,7 +76,8 @@ async fn test_server() {
         logger.clone(),
     ));
     lookup_data.refresh().await.unwrap();
-    let lookup_factory = LookupFactory::create(lookup_data, logger.clone()).unwrap();
+    let lookup_factory =
+        LookupFactory::new_boxed_extension_factory(lookup_data, logger.clone()).unwrap();
```

Comment:
`expect` instead of `unwrap`? And in a few other places too.

Reply:
done (and in the other places :))